store/sqlstore/post_store.go

```diff
@@ -1048,35 +1048,34 @@ func (s *SqlPostStore) AnalyticsPostCountsByDay(teamId string) (model.AnalyticsR
 	return rows, nil
 }
 
-func (s *SqlPostStore) AnalyticsPostCount(teamId string, mustHaveFile bool, mustHaveHashtag bool) store.StoreChannel {
-	return store.Do(func(result *store.StoreResult) {
-		query :=
-			`SELECT
+func (s *SqlPostStore) AnalyticsPostCount(teamId string, mustHaveFile bool, mustHaveHashtag bool) (int64, *model.AppError) {
+	query :=
+		`SELECT
 			    COUNT(Posts.Id) AS Value
```

Comment:
nit: we should keep the query identation, that means remove a tab from each query line too, not only for the first line.